pkg/manager/manager_test.go

```diff
@@ -452,7 +536,8 @@ var _ = Describe("manger.Manager", func() {
 				Expect(err).To(MatchError(ContainSubstring("expected error")))
 			})
 			It("should return an error if namespace not set and not running in cluster", func() {
-				m, err := New(cfg, Options{LeaderElection: true, LeaderElectionID: "controller-runtime"})
+				leaderElectOpt := true
+				m, err := New(cfg, Options{LeaderElection: &leaderElectOpt, LeaderElectionID: "controller-runtime"})
```

Comment:
Somewhat nitpicky, but if we import the `utils/pointer` image anyways, why not use `pointer.Boolptr` everywhere? It makes the code IMHO easier to read as you don't have to go back to check what the value of `leaderElectOpt` is

Reply:
Agree. Changed